src/commands/BootstrapCommand.js

```diff
@@ -28,8 +28,10 @@ export default class BootstrapCommand extends Command {
    * @param {Function} callback
    */
   bootstrapPackages(callback) {
-    this.filteredGraph = PackageUtilities.getPackageGraph(this.filteredPackages);
     this.logger.info(`Bootstrapping ${this.filteredPackages.length} packages`);
+    this.batchedPackages = this.flags.toposort
```

Comment:
Oh, boy.  Yeah, I appreciate that making it opt-in for bootstrap is consistent.

I guess a repo that _does_ need it can just add it to `lerna.json`.